source/globalCommands.py

```diff
@@ -4888,6 +4891,69 @@ def script_toggleApplicationsVolumeAdjuster(self, gesture: "inputCore.InputGestu
 	def script_toggleApplicationsMute(self, gesture: "inputCore.InputGesture") -> None:
 		appsVolume._toggleAppsVolumeMute()
 
+	@script(
+		# Translators: Describes a command.
+		description=_("""Mute or unmute the speech coming from the remote computer"""),
+		category=SCRCAT_REMOTE,
+	)
+	def script_toggle_remote_mute(self, gesture):
+		remoteClient.remoteClient.toggleMute()
+
+	@script(
+		gesture="kb:control+shift+NVDA+c",
+		category=SCRCAT_REMOTE,
+		# Translators: Documentation string for the script that sends the contents of the clipboard to the remote machine.
+		description=_("Sends the contents of the clipboard to the remote machine"),
+	)
+	def script_push_clipboard(self, gesture):
+		remoteClient.remoteClient.pushClipboard()
+
+	@script(
+		# Translators: Documentation string for the script that copies a link to the remote session to the clipboard.
+		description=_("""Copies a link to the remote session to the clipboard"""),
+		category=SCRCAT_REMOTE,
+	)
+	def script_copy_link(self, gesture):
+		remoteClient.remoteClient.copyLink()
+		# Translators: A message indicating that a link has been copied to the clipboard.
+		ui.message(_("Copied link"))
+
+	@script(
+		gesture="kb:alt+NVDA+pageDown",
+		category=SCRCAT_REMOTE,
+		# Translators: Documentation string for the script that disconnects a remote session.
+		description=_("""Disconnect a remote session"""),
+	)
+	@gui.blockAction.when(gui.blockAction.Context.SECURE_MODE)
+	def script_disconnectFromRemote(self, gesture):
+		if not remoteClient.remoteClient.isConnected:
+			# Translators: A message indicating that the remote client is not connected.
+			ui.message(_("Not connected."))
+			return
+		remoteClient.remoteClient.disconnect()
+
+	@script(
+		gesture="kb:alt+NVDA+pageUp",
+		# Translators: Documentation string for the script that invokes the remote session.
+		description=_("""Connect to a remote computer"""),
+		category=SCRCAT_REMOTE,
+	)
+	@gui.blockAction.when(gui.blockAction.Context.MODAL_DIALOG_OPEN)
+	@gui.blockAction.when(gui.blockAction.Context.SECURE_MODE)
+	def script_connectToRemote(self, gesture):
+		if remoteClient.remoteClient.isConnected() or remoteClient.remoteClient.connecting:
+			return
+		remoteClient.remoteClient.doConnect()
+
+	@script(
+		# Translators: Documentation string for the script that toggles the control between guest and host machine.
+		description=_("Toggles the control between guest and host machine"),
+		category=SCRCAT_REMOTE,
+		gesture="kb:f11",
```

Comment:
As other reviewers, I think that this gesture shouldn't be used here since it's used by other apps. NVDA+f11 seems to be unused and may be appropriate.